.github/workflows/helm.yaml

```diff
@@ -78,5 +78,6 @@ jobs:
           CR_SKIP_EXISTING: true
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
           CR_RELEASE_NAME_TEMPLATE: "helm-chart-{{ .Version }}"
+          CR_GENERATE_RELEASE_NOTES: true
```

Comment:
I don't think Go template language is available here and I also don't know if the chart version to be released can already be found in the workflow's facts.

Reply:
This value is not getting template: https://github.com/helm/chart-releaser/blob/main/pkg/releaser/releaser.go#L253-L263

We could either determine the version from the Chart.yaml our own or file a PR adding the ability to template the release notes file name, so `CR_RELEASE_NAME_TEMPLATE: "helm-chart-{{ .Version }}"` works.

Reply:
@cpanato As you happen to be a member of `helm/chart-releaser`: What's your opinion on that?

Reply:
sorry for the delay
I think this will not work properly, because that will catch all commits and not the commits for the chart.
that feature was built to run in a mono repo only for charts

Reply:
Yeah, that's why I pointed to `CR_RELEAE_NOTES_FILE`.

`charts/ingress-nginx/changelog/helm-chart-{version}.md` is normally created manually beforehand, so all we need to do is point `chart-releaser` to the semi-manually crafted changelog.

Right now we have those files already lying around in `charts/ingress-nginx/changelog/` and it would be awesome to also include the according `helm-chart-{version}.md` in GitHub releases.

Reply:
For this to work we would need to either collect the chart version as an output so we can use it in the path or make `chart-releaser` capable of also using Go template in the `CR_RELEAE_NOTES_FILE` environment variable.

Reply:
yes, that makes sense

Reply:
@jessebot Could you either figure out a way to get the version to be released into the GitHub action outputs and expand the `CR_RELEAE_NOTES_FILE` using it or propose a PR to `helm/chart-releaser` so one can use Go template as it's already possible for `CR_RELEASE_NAME_TEMPLATE`?